runner/__init__.py

```diff
@@ -192,6 +203,9 @@ def make_argument_parser():
     parser.add_argument("-n", "--times", dest="times", type=int, help="run this many times (default is forever)")
     parser.add_argument("-H", "--halt-after", dest="halt_after", action="store_const", const=True,
                         help="Call the halt task after runner finishes (never called if -n is not set).")
+    parser.add_argument("--syslog", dest="syslog", action="store_const", const=True, help="send messages to syslog")
+    parser.add_argument("--syslog-address", dest="syslog_address", default=get_syslog_address(),
```

Comment:
This should be --syslog-device. SyslogHandler doesn't support strings for IP adresses, it has to be a tuple of (address/hostname, port).